src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -1577,11 +1577,11 @@ public void TestLiveQueryBlocksClose()
                 }
 
                 doc1Listener.WaitForResult(TimeSpan.FromSeconds(20));
-                otherDb.Invoking(d => d.Dispose())
-                    .Should().Throw<CouchbaseLiteException>("because the live query is still active");
+                //otherDb.Invoking(d => d.Dispose())
+                //    .Should().Throw<CouchbaseLiteException>("because the live query is still active");

```

Comment:
I assume this exception will no longer throw because live queries and replications should all stopped before database closing.